### DISCUSSION TITLE
Imports on install

### DISCUSSION THREAD

**Discussion by Neo-Ryo** [adonisjs/core#4337] [Help] [answered]
I've watch a talk about AdonisJS a few days ago and was very enthusiastic about trying it out. So I went for the slim version and the api version yarn/pnpm create I was surprised that some imports are incorrect. It seems like the "Config" import from jappa is wrong, and also the "processCliArgs" and "RunnerHooksHandler" from `test.ts` (and would be hard to fix for a neophyte). So am I missing something? I humbly ask for a fix and if I can be of any help do not hesitate to reach me out.
![Screenshot from 2024-01-08 22-47-28](https://github.com/adonisjs/core/assets/61702850/77aad531-3178-42e9-956a-14a87ecda9b7)
![Screenshot from 2024-01-08 22-47-41](https://github.com/adonisjs/core/assets/61702850/a2219c5a-37f9-45e9-b97d-ef61496b5b2c)

**Comment by benjibenoit** [comment, marked as answer]
Hi,
What version of japa/adonisjs are you using?
I had the same error because japa was not the same version.
If you're on `"@adonisjs/core": "^5.9.0"` try `"@japa/runner": "^2.5.1"`.

**Reply by Neo-Ryo** [reply]
Thx for the reply, I moved on v6 and the issue seems to be gone.
